src/screens/Home/Start.js

```diff
@@ -98,9 +108,10 @@ render(<App />);
           round={{ corner: 'top' }}
           border={[{ side: 'top' }, { side: 'vertical' }]}
           pad={{ top: 'medium' }}
+          background={{ color: isDarkMode ? '#121212' : '#FFFFFF' }}
```

Comment:
Do we need a background color here?
If we do instead of hard coding hex values here can we use values from the theme?